publish/models/version.py

```diff
@@ -19,6 +19,53 @@ def _get_default_version() -> str:
     return Version.make_version()
 
 
+class Contributor(models.Model):
+    class RoleType(models.TextChoices):
+        Author = "Author"
+        Conceptualization = "Conceptualization"
+        ContactPerson = "ContactPerson"
+        DataCollector = "DataCollector"
+        DataCurator = "DataCurator"
+        DataManager = "DataManager"
+        FormalAnalysis = "FormalAnalysis"
+        FundingAcquisition = "FundingAcquisition"
+        Investigation = "Investigation"
+        Maintainer = "Maintainer"
+        Methodology = "Methodology"
+        Producer = "Producer"
+        ProjectLeader = "ProjectLeader"
+        ProjectManager = "ProjectManager"
+        ProjectMember = "ProjectMember"
+        ProjectAdministration = "ProjectAdministration"
+        Researcher = "Researcher"
+        Resources = "Resources"
+        Software = "Software"
+        Supervision = "Supervision"
+        Validation = "Validation"
+        Visualization = "Visualization"
+        Funder = "Funder"
+        Sponsor = "Sponsor"
+        StudyParticipant = "StudyParticipant"
+        Other = "Other"
+
+    name = models.TextField(default=str)
```

Comment:
What's the intention of `default=str`?

Reply:
I think my intention was to provide a default value of an empty string for those fields. But they should always be provided by the client so I think I can remove them.